content-src/asrouter/asrouter-content.jsx

```diff
@@ -286,15 +276,25 @@ export class ASRouterUISurface extends React.PureComponent {
     const { message } = this.state;
     if (FIRST_RUN_TEMPLATES.includes(message.template)) {
       return (
-        <FirstRun
+        <ImpressionsWrapper
```

Comment:
Needed to add impressions since there's frequency capping involved. Should I only do it for new tabs (i.e. not on about:welcome)